install_config/install/advanced_install.adoc

```diff
@@ -680,6 +680,217 @@ openshift_examples_modify_imagestreams=true
+.. Verify that the route was created:
++
+----
+# oc get routes
+NAME            HOST/PORT                                     PATH           SERVICES   PORT   TERMINATION   WILDCARD
+heketi-glusterfs-route  heketi-glusterfs-default.cloudapps.example.com  heketi-glusterfs <all>             None
+----
++
+[NOTE]
+====
+The name for the route will be `heketi-glusterfs-route` unless the default
+`glusterfs` value was overridden using the `openshift_glusterfs_storage_name`
+variable in the inventory file.
+====
+
+.. Curl the route to verify it works correctly:
++
+----
+# curl http://heketi-glusterfs-default.cloudapps.example.com/hello
+Hello from Heketi.
+----
+
+After successful installation,
+xref:../storage_examples/gluster_dynamic_example.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[dynamic provisioning] of GlusterFS volumes can occur by
+xref:../storage_examples/gluster_dynamic_example.html#create-a-pvc-ro-request-storage-for-your-application[creating a PVC to request storage].
+
+////
+[[advanced-install-dedicated-glusterfs-persistent-storage]]
+==== Configuring Dedicated GlusterFS Persistent Storage
+
+TODO
+////
+
+[[advanced-install-configuring-openshift-container-registry]]
+=== Configuring the OpenShift Container Registry
+
+Additional configuration options are available at installation time for the
+xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[OpenShift Container Registry].
+
+If no registry storage options are used, the default {product-title} registry is
+ephermal and all data will be lost if the pod no longer exists. {product-title}
+also supports a single node NFS-backed registry, but this option lacks
+redundancy and reliability compared with the GlusterFS-backed option.
+
+[[advanced-install-containerized-glusterfs-backed-registry]]
+==== Configuring a Containerized GlusterFS-Backed Registry
+
+Similar to
+xref:advanced-install-containerized-glusterfs-persistent-storage[configuring containerized GlusterFS for persistent storage], GlusterFS storage can be
+configured and deployed for an OpenShift Container Registry during the initial
+installation of the cluster to offer redundant and more reliable storage for the
+registry.
+
+[IMPORTANT]
+====
+See
+xref:../../install_config/install/prerequisites.adoc#prereq-containerized-glusterfs-considerations[Containerized
+GlusterFS Considerations] for specific host preparations and prerequisites.
+====
+
+Configuration of storage for an OpenShift Container Registr is very similar to
+configuration for regular GlusterFS persistent storage in that it can be either
+containerized or non-containerized. The following exceptions and additions
+apply:
+
+. In your inventory file, add `glusterfs_registry` to the `[OSEv3:children]` section
+to enable the `[glusterfs_registry]` group:
++
+----
+[OSEv3:children]
+masters
+nodes
+glusterfs_registry
+----
+
+. Add the following role variable in the `[OSEv3:vars]` section:
++
```

Comment:
Here it is "**in** the section"; previous step is "**to** the section". Is this intentional?